src/truffle.ts

```diff
@@ -0,0 +1,518 @@
+          args: {
+            name: "network"
+          }
+        },
+        {
+          name: ["--fetch-external", "-x"],
+          description: "Allows the debugger to download source from source verification services to debug transactions involving external contracts. When used, a transaction hash is required. May be abbreviated -x.",
+        },
+        {
+          name: ["--compile-tests"],
+          description: "Allows the debugger to compile Solidity test contracts. Implies --compile-all.",
+          exclusiveOn: ["--compile-all", "--compile-none"]
+        },
+        {
+          name: ["--compile-all"],
+          description: "Forces the debugger to recompile all contracts, even when it would otherwise judge doing so unnecessary. Compilation results are not saved.",
+          exclusiveOn: ["--compile-tests", "--compile-none"]
+        },
+        {
+          name: ["--compile-none"],
+          description: "Forces the debugger not to recompile contracts, even when it would otherwise judge it necessary. This option is dangerous and may cause errors. Please only use this if you are sure a recompilation is not necessary.",
+          exclusiveOn: ["--compile-tests", "--compile-all"]
+        },
+      ],
+    },
+    {
+      name: "deploy",
+      description: "Alias for migrate. See migrate for details.",
+    },
+    {
+      name: "develop",
+      description: "Open a console with a development blockchaine.",
+      options: [
+        {
+          name: ["--log"],
+          description: " Start/Connect to a Truffle develop session and log all RPC activity. See the Log RPC Activity docs for more information about using this option.",
+          args: {
+            name: "network"
+          }
+        },
+        {
+          name: ["--require", "-r"],
+          description: "Preload console environment from required JavaScript file.",
+          args: {
+            name: "file",
+            description: "The default export must be an object with named keys that will be used to populate the console environment. For example, if your JavaScript is module.exports = { desert: \"yes please!\" } then breakfast will be available in the console with the value \"yes please!.\"",
+          }
+        },
+        {
+          name: ["--require-none"],
+          description: "Do not load any user-defined JavaScript into the console environment. This option takes precedence over --require, -r, and values provided for console.require in your project's truffle-config.js.",
+        },
+      ]
+    },
+    {
+      name: "exec",
+      description: "Execute a JS module within the Truffle environment.",
+      args: {
+        name: "script.js",
+        description: "JavaScript file to be executed. Can include path information if the script does not exist in the current directory. (required)",
+        isOptional: false,
+        generators: {
+          template: "filepaths"
+        }
```

Comment:
A simpler way to achieve this is:
```suggestion
        template: "filepaths"
```